kratos/includes/oriented_bounding_box.h

```diff
@@ -134,7 +134,8 @@ class KRATOS_API(KRATOS_CORE) OrientedBoundingBox
      */
     OrientedBoundingBox(
         const GeometryType& rGeometry,
-        const double BoundingBoxFactor
+        const double BoundingBoxFactor,
+        const bool BuildFromBoundingBox = true
```

Comment:
According to the description of the PR, shouldn't this be `false` by default?

Reply:
No, true is the the current behavior, false is the new one